_source/_posts/2024-02-29-3pc.md

```diff
@@ -0,0 +1,65 @@
+
+Today, the drawbacks to users' security and privacy from 3rd party cookie implementations outweigh their benefits so much that all major browsers are phasing them out. Firefox, Edge, and Safari have been blocking 3rd party cookies for years, and Chrome is now phasing them out in 2024. 
+
+If your code uses Okta features that rely on 3rd party cookies, this means that you'll need to make some changes to keep the identity experience working as intended. 
+
+
+## Does your application use 3rd party cookies? 
+
+Most code using Okta doesn't use 3rd party cookies, and won't be affected by the deprecation. Here are the design patterns in which Okta uses 3rd party cookies. If your application is in one of these categories, please test its behavior with 3rd party cookie deprecation. 
+
+Okta uses cookies to let applications introspect and extend user sessions. Cookies aren't required for basic login functionality. 
+
+### Web applications that rely on the Okta session for user context.  
+
+If an application hosted on your domain (`mycompanyapp.com`) redirects to your Okta subdomain (`mycompany.okta.com`) for login and then returns users to your own domain, 3rd party cookie restrictions will limit how your app can introspect or extend the Okta session. 
+
+
+### Customer-hosted Okta Sign-In Widget and customer-built login applications
+
+If you're hosting your own sign-in experience on a separate top level domain from your main app, you may be using 3rd party cookies. You might be hosting your own sign-in experience by cloning the Okta Sign-In Widget from GitHub or installing it from NPM to embed in your application, or you might have built your own custom sign-in using Okta's APIs. 
+
+If your sign-in experience is hosted on the same top-level domain as your application, 3rd party cookie deprecation won't affect its behavior. 
+
+If the sign-in experience and app are on different top-level domains, 3rd party cookie deprecation will break its ability to introspect and extend sessions, because these features use cookies. Authentication will still be possible, and tokens will still be returned, becasue these features do not rely on cookies. 
+
+### "Remember me" Features
+
+"Remember Last Used Factor" (RLUF), "Keep me signed in" (OIE) and "Remember me" (Classic) will not function when the login application is on a 3rd party domain.  
+
+## When will this affect you? 
+
+Google has made an exemption for Okta's 3rd party cookies until the end of 2024. However, you can set Chrome's flags to simulate how the browser will treat Okta's 3rd party cookies after that exemption ends. 
+
+### Test your application today!
+
+To simulate how Chrome will treat Okta's 3rd party cookies in 2025 and beyond, follow [Okta's help center guide](https://support.okta.com/help/s/article/deprecation-of-3rd-party-cookies-in-google-chrome?language=en_US). 
```

Comment:
Update the link copy to be more descriptive, perhaps including the article's title.